addons/event/static/src/client_action/event_barcode.js

```diff
@@ -84,16 +95,7 @@ export class EventScanView extends Component {
                 type: "ir.actions.act_window",
                 name: _t("Events"),
                 res_model: "event.event",
-                views: [
-                    [false, "kanban"],
-                    [false, "calendar"],
-                    [false, "list"],
-                    [false, "gantt"],
-                    [false, "form"],
-                    [false, "pivot"],
-                    [false, "graph"],
-                    [false, "map"],
-                ],
+                views: this.getEventAvailableViews(),
                 target: "main",
             });
```

Comment:
I think that we could just call the event action instead which will have the correct views based on community/enterprise.

Something like this should do the trick:
```js
this.actionService.doAction("event.action_event_view", {
    clearBreadcrumbs: true,
});
```

Reply:
@jeh-odoo I know but when i see this comment of the author `// define action from scratch instead of using existing 'action_event_view' to avoid
            // messing with menu bar` then i decided to not do that. You know to keep think as it use to be

Reply:
wait is `clearBreadcrumbs` help that

Reply:
yes, normally `clearBreacrumbs` should update the UI at some point and display the menu correctly if I remember correctly

Reply:
Nice it does do the trick, thank you very much, updated please r+